BacnetException.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace System.IO.BACnet
+{
+    public class BacnetException : Exception

```

Comment:
Forgot to save the file in csproj? Create can also create a subnamespace Exception in which you can store each class in separate file. It's ok for now but we don't want to end up with too big file when we start adding new ones.